src/screens/DetailScreen.js

```diff
@@ -115,7 +116,11 @@ export const DetailScreen = ({ navigation, route }) => {
   });
 
   return (
-    <Query query={getQuery(query)} variables={{ id: queryVariables.id }} fetchPolicy={fetchPolicy}>
+    <Query
+      query={getQuery(query)}
+      variables={{ id: queryVariables.id, date: date }}
```

Comment:
could be `variables={{ id: queryVariables.id, date }}`

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/4cb2d742b6a72d12ab35b2c1b56f9fe5fe293ff2